app/views/close_reasons/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<% content_for :title, "Close Reasons" %>
+
+<h1>Close Reasons</h1>
+
+<p class="is-lead">
+  This page lists all active and inactive close reasons and allows you to edit them.
+  Moderators can edit close reasons for this community, administrators for all.
+</p>
+<p>The column <em>require link?</em> has a "yes" value if you need to add a link to another post in order to use the close reason.</p>
+
+<% if admin? %>
+  <div class="button-list is-gutterless has-margin-bottom-4">
+    <%= link_to "current community", "?global=0", class: "button is-muted is-outlined #{(params[:global]=='0' || params[:global].nil?) ? 'is-active' : ''}" %>
+    <%= link_to "global", "?global=1", class: "button is-muted is-outlined #{(params[:global]=='1') ? 'is-active' : ''}" %>
+  </div>
+<% end %>
+
+<table class="table is-with-hover">
```

Comment:
Not one for this PR, but I'd suggest renaming this class to `has-hover-effect`. `is-with` sounds really awkward.